src/main/java/tars/TaskList.java

```diff
@@ -0,0 +1,199 @@
+package tars;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+/**
+ * Class represents a taskList which handles the list of the tasks being added by the User
+ * Handles operations of adding tasks to list based on input of Task type
+ *
+ * @author csk
+ * @version 1
+ */
+public class TaskList {
+    private static final String LINE = "    _____________________________________________";
+    private ArrayList<Task> taskList;
+
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+    public TaskList() {
+        this.taskList = new ArrayList<Task>();
+    }
+
+    public ArrayList<Task> getList() {
+        return this.taskList;
+    }
+
+    /**
+     * Adds task to list based on input given or marks/unmarks task based on action required
+     * Calls respective method to add Task based on type identified of task stated in input
+     *
+     * @param task
+     * @param entry
+     */
+    public String addTask(String[] task, String entry) {
+        String result = "";
+        if (task[0].equals("mark")) {
+            Integer index = Integer.parseInt(task[task.length - 1]); //convert string format of number to Integer
+            taskList.get(index - 1).mark(); //marking TASK as done
+
+            result = LINE + "\n" + "    Nice! I've marked this task as done:" + taskList.get(index - 1) + "\n" + LINE;
+        } else if (task[0].equals("unmark")) {
+            Integer index = Integer.parseInt(task[task.length - 1]); //convert str format of number to Integer
+            taskList.get(index - 1).unmark(); //unmarking TASK as not done
+
+            result = LINE + "    OK, I've marked this task as not done yet:" + taskList.get(index - 1) + "\n" + LINE;
+        } else if (task[0].equals("todo")) {
+            result = this.addToDos(task, entry);
```

Comment:
I am wondering why ``addToDos`` ends with an **s** while ``addDeadline`` and ``addEvent`` don't?